packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -301,6 +301,9 @@ abstract contract StandardBridge {
         require(_to != address(this), "StandardBridge: cannot send to self");
         require(_to != address(MESSENGER), "StandardBridge: cannot send to messenger");
 
+        // If the _emit function is overri theden it will emit legacy events before emitting the
+        // current Bridge event.
+        _emitETHBridgeFinalized(_from, _to, _amount, _extraData);
         emit ETHBridgeFinalized(_from, _to, _amount, _extraData);
```

Comment:
Hmm this reads kind of strange because its basically the same thing - should both events be inside the function in this case?